server/controllers/stock/reports/stock/stock_sheet.js

```diff
@@ -32,7 +32,9 @@ async function stockSheetReport(req, res, next) {
 
     const data = { inventory, depot };
 
-    data.rows = rows.movements.sort((x, y) => x.date - y.date);
+    // data.rows = rows.movements.sort((x, y) => x.date - y.date);
```

Comment:
I don't think you meant to comment out this line.  If I uncomment it, the sort works great.  If left commented out no sorting happens in the report regardless of which option I choose for the sorting.

Reply:
I was able to verify that the sorting flag works (if uncommented).